File: PHP/8php 3/reverseSeq.php
<?php
/*
|--------------------------------------------------------------------------
| Reversed sequence
|--------------------------------------------------------------------------
|
*/

function reverseSeq ($n) {
   $res = [];
   for ($i=1; $i <= $n; $i++) { 
      array_push($res, $i);
   }
   return array_reverse($res);
};


function reverseSeq ($n) {
   for ($i = $n; $i > 0; $i--) {
      $result[] = $i ;
   }
   return $result;
};


function reverseSeq ($n) {
   return range($n, 1);
};


function reverseSeq ($n) {
   while($n>=1) {
      $arr[] += $n--;
   }
   return $arr;
};


print_r(reverseSeq(5));

File: Python/8py 2/reverse_seq.py
"""
|--------------------------------------------------------------------------
| Reversed sequence
|--------------------------------------------------------------------------
|
"""

def reverse_seq(n):
   res = []
   for i in range(n):
      res.append(n)
      n -= 1
   return res


def reverseseq(n):
   return range(n, 0, -1)


def reverseseq(n):
   return [x for x in range(n,0,-1)]


def reverse_seq(n):
   l = [x for x in range(1,n+1)]
   l.reverse()
   return l


def reverse_seq(n): 
   return list(reversed(range(1,n+1)))


print(reverse_seq(5),[5,4,3,2,1])